todo_app/app.py

```diff
@@ -1,11 +1,18 @@
-from flask import Flask
+from flask import Flask, redirect, render_template, request
+from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.get('/')
 def index():
-    return 'Hello World!'
+   items = get_items()
+   return render_template('index.html',items=items)
+
+@app.post('/')
+def add_new_item():
+   new_item = add_item(request.form["title"])
+   return redirect("/", code=302)
```

Comment:
Nice, the default status code for redirect is 302 anyway, but I'm a fan of making it explicit that this is a standard redirect (non-permanent, in the sense I don't want browsers to skip the original post request!)

This sent me down a brief rabbit-hole of whether 302 is really the correct one for this pattern which led me to [the Post/Redirect/Get named pattern](https://en.wikipedia.org/wiki/Post/Redirect/Get) which tends to [prefer 303 over 302](https://serverfault.com/questions/391181/examples-of-302-vs-303), but it seems like a fairly minor/unimportant difference to me!